lib/link.ts

```diff
@@ -37,6 +38,20 @@ export const handleLink = (url: string) => {
 
     console.log("Navigating to", fullUrl);
 
+    const lnurlValue = lnurl.findLnurl(fullUrl);
+    if (lnurlValue) {
+      const lnurlDetails = await lnurl.getDetails(lnurlValue)
+
+      if (lnurlDetails.tag === "withdrawRequest") {
+        router.push({
+          pathname: "/withdraw",
+          params: {
+            url: fullUrl,
+          },
+        });
+      }
+    }
+
```

Comment:
Could've added a check for "payRequest" but we don't strip `lightning:` here, should we make `getDetails` handle it?

Reply:
Sorry I mean make `findLnurl` return a lightning address as well, currently it doesn't return `hello@getalby.com` when `lightning:hello@getalby.com` is passed

Reply:
Yeah, but let's do it as a follow-up. Let's also add some tests for this lib function.

Reply:
https://github.com/getAlby/go/issues/162